--- system-config-date-1.9.67/system-config-date.desktop.in~	2011-10-18 13:49:29.000000000 +0300
+++ system-config-date-1.9.67/system-config-date.desktop.in	2012-03-03 13:37:50.852903093 +0200
@@ -3,7 +3,7 @@
 _GenericName=Set Date & Time
 _X-GNOME-FullName=System-Config-Date (Set Date & Time)
 _Comment=Change system date and time
-Categories=System;Settings;
+Categories=GTK;System;Settings;
 Icon=system-config-date
 Exec=/usr/bin/system-config-date
 Type=Application
